pennylane/measurements.py

```diff
@@ -236,7 +241,7 @@ def shape(self, device=None):
             return shape
 
         # Then: handle return types that require a device; no shot vector
-        if device is None and self.return_type in (Probability, State, Sample):
+        if device is None and self.return_type in (Probability, State, Sample, Counts):
```

Comment:
This method is for determining the shape of the measurement process. Does this make sense for `Counts`?